src/host/screenInfo.cpp

```diff
@@ -2035,10 +2035,16 @@ const SCREEN_INFORMATION& SCREEN_INFORMATION::GetMainBuffer() const
 
     const FontInfo& existingFont = GetCurrentFont();
 
+    // The buffer needs to be initialized with the standard erase attributes,
+    // i.e. the current background color, but with no meta attributes set.
+    auto initAttributes = GetAttributes();
+    initAttributes.SetExtendedAttributes(ExtendedAttributes::Normal);

```

Comment:
I feel like this pattern (SetExtendedAttributes to Normal, set Meta to 0) is common enough that perhaps there should just be a .SetStandardErase() method on the Attributes that does both of these instead of having this pattern everywhere.

Reply:
Very happy to do this. Makes things a lot neater. Added in commit 786cc365b1c75ce8192496056c570eef12c39617